public/app/plugins/datasource/cloudwatch/components/VariableQueryEditor/VariableQueryEditor.tsx

```diff
@@ -122,6 +133,18 @@ export const VariableQueryEditor = ({ query, datasource, onChange }: Props) => {
           inputId={`variable-query-region-${query.refId}`}
         />
       )}
+      {hasAccountIDField &&
+        accountState.value &&
```

Comment:
The linter gets mad at me if I remove this line 😞 . Also, if I check for `accountState.value?.length` instead of `accountState.value?.length > 0` on the next line it prints out a 0 when the datasource doesn't have multiple accounts.

Reply:
Lovely quirks of Javascript 🥰 Definitely keep the length > 0 then, you can ignore the suggestion :)